test-src/manageamiset_test.js

```diff
@@ -0,0 +1,58 @@
+var slugid = require('slugid');
+var assume = require('assume');
+var debug = require('debug')('test');
+var _ = require('lodash');
+var mock = require('./mock-amisets');
+var base = require('taskcluster-base');
+
+var makeAmiSet = mock.makeAmiSet;
+
+var main = require('../lib/main');
+var helper = require('./helper');
+
+describe('AMI Set api', () => {
+
+  var id = slugid.nice();
+  var amiSetDefinition = makeAmiSet();
+  var amiSetChanged = _.clone(amiSetDefinition);
```

Comment:
Without an update endpoint, this variable is not really useful.  For the worker type tests, we have this changed variable where we basically create a second version of the tested definition, with one value changed.  We then use the update method to verify that this expected value matches the one out of the api/table storage.  For now, we should either delete this variable or we should have the update logic implemented.